src/quo2/components/small_option_card/__tests__/small_option_card_component_spec.cljs

```diff
@@ -0,0 +1,51 @@
+(ns quo2.components.small-option-card.--tests--.small-option-card-component-spec
```

Comment:
this can be in the name space

`quo2.components.small-option-card..component-spec`

also I just noticed you need to adjust the name spaces in quo2, they should match figma section and component name..

i.e in this case

quo2.components.onboarding-small-option-card.view
quo2.components.onboarding-small-option-card.style
quo2.components.onboarding-small-option-card.component-test

Reply:
Oh, thanks! I tried to follow the design's :sweat_smile: 

Solved in 3bb7d951aedfe1be12697f6546b774f4bcb01c49

I moved them, except the last one suggested:
`quo2.components.onboarding-small-option-card.component-test`
Since there's no other test namespace ending with `-test` in the ns `quo2.core-spec`, they  have the suffix `-spec`:

```clojure
(:require [quo2.components.banners.banner.component-spec]
            [quo2.components.buttons.--tests--.buttons-component-spec]
            [quo2.components.counter.--tests--.counter-component-spec]
            [quo2.components.onboarding.small-option-card.component-spec]  ;; <- I added this one
            [quo2.components.dividers.--tests--.divider-label-component-spec]
            [quo2.components.drawers.action-drawers.component-spec]
            [quo2.components.drawers.permission-context.component-spec]
            [quo2.components.markdown.--tests--.text-component-spec]
            [quo2.components.selectors.--tests--.selectors-component-spec]
            [quo2.components.selectors.filter.component-spec]
            [quo2.components.record-audio.record-audio.--tests--.record-audio-component-spec]
            [quo2.components.record-audio.soundtrack.--tests--.soundtrack-component-spec])
```

Will the new component tests added not be inside a `--test--` namespace?

Reply:
Whoops, I meant -spec 😑